wen-restart/src/wen_restart.rs

```diff
@@ -2516,7 +2634,23 @@ mod tests {
             find_heaviest_fork(
                 LastVotedForkSlotsFinalResult {
                     slots_stake_map,
-                    total_active_stake: 900,
+                    epoch_info_vec: vec![
+                        LastVotedForkSlotsEpochInfo {
+                            epoch: 0,
+                            total_stake: 1000,
+                            total_active_stake: 900,
+                        },
+                        LastVotedForkSlotsEpochInfo {
+                            epoch: 1,
+                            total_stake: 1000,
+                            total_active_stake: 900,
+                        },
+                        LastVotedForkSlotsEpochInfo {
+                            epoch: 2,
+                            total_stake: 1000,
+                            total_active_stake: 900,
+                        },
+                    ],
```

Comment:
Would be good to see a test where the stakes across a boundary pick two different forks and see we halt

Reply:
test added.